tripy/tests/test_function_registry.py

```diff
@@ -474,7 +474,7 @@ def func(n: Sequence[Union[int, float]]) -> int:
     "typ, expected",
     [
         (tp.types.TensorLike, "Union[tripy.Tensor, numbers.Number]"),
-        (tp.types.ShapeLike, "Union[tripy.Shape, Sequence[Union[int, tripy.ShapeScalar]]]"),
+        (tp.types.ShapeLike, "Sequence[Union[int, tripy.DimensionSize]]"),
```

Comment:
shouldn't `tripy.DimensionsSize` be an instance of `TensorLike` since it inherits from the `Tensor` class?

Reply:
Yes, it is, since `TensorLike` includes `Tensor` (and therefore all children of `Tensor`).